.readthedocs.yaml

```diff
@@ -0,0 +1,6 @@
+version: 2
+
+python:
+  version: "3.7"
+  install:
+    - requirements: requirements/py37-django32.txt
```

Comment:
updating to the `.readthedocs.yaml` I use in my other projects - using latest python, not a test requirements file